### PR TITLE
Adding multipoint_navigation_receiver,wit-imu,and orb_slam2_with_maps-odom to documentation index for distro

### PR DESCRIPTION
**Pull request by SuperAruba**
Adding orb_slam2_with_maps-odom to documentation index for distro

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Please edit the title of the pull request to describe it's content since it will became the message of the squash merged commit.

**Comment by SuperAruba** [comment]
Is there any problem with this title, please？

**Comment by hidmic** [comment]
@SuperAruba the current title mentions only one package of all packages affected by this patch.

**Comment by SuperAruba** [comment]
Is this title OK?

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -8108,11 +8108,11 @@ repositories:
   multipoint_navigation_receiver:
     doc:
       type: git
-      url: https://github.com/SuperAruba/multipoint_navigation_receiver.git
+      url: https://github.com/Inspur-ROS/multipoint_navigation_receiver.git
```
Comment (ivanpauno) [inline kinetic/distribution.yaml:8111]: why is being the organization updated to a fork?

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -9498,6 +9498,16 @@ repositories:
       url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
       version: master
     status: maintained
+  orb_slam2_with_maps-odom:
+    doc:
+      type: git
+      url: https://github.com/Inspur-ROS/ORB_SLAM2_WITH_MAPS-ODOM.git
```
Comment (ivanpauno) [inline kinetic/distribution.yaml:9504]: is this a ROS package?
I don't see a package.xml here

Comment (ivanpauno) [reply kinetic/distribution.yaml:9504]: this package also seems to be a fork

File: kinetic/distribution.yaml [hunk 3]
```diff
@@ -17531,6 +17541,16 @@ repositories:
       url: https://github.com/clearpathrobotics/wireless.git
       version: master
     status: maintained
+  wit-imu-driver:
```
Comment (ivanpauno) [inline kinetic/distribution.yaml:17544]: The repo doesn't have a top level license file, could you add one?
Same question about being a fork.

File: kinetic/distribution.yaml (hunk 1 above)
Comment (SuperAruba) [reply kinetic/distribution.yaml:8111]: This is our own organization for putting multiple codes together,and Fork is for easy maintenance

File: kinetic/distribution.yaml (hunk 2 above)
Comment (SuperAruba) [reply kinetic/distribution.yaml:9504]: This package is based on orb_slam2 demo,we should cd ORB_SLAM2_With_Maps-Odom/Examples/ROS/ORB_SLAM2,there is a manifest.xml.

File: kinetic/distribution.yaml (hunk 3 above)
Comment (SuperAruba) [reply kinetic/distribution.yaml:17544]: OK,I have notified my colleagues to add, and I will reply to you after adding.

Comment (SuperAruba) [reply kinetic/distribution.yaml:17544]: Now,It has a license

File: kinetic/distribution.yaml (hunk 1 above)
Comment (ivanpauno) [reply kinetic/distribution.yaml:8111]: Ok, considering that the same people seem to be the maintainers of both the fork and the original repo, this looks fine to me.

File: kinetic/distribution.yaml (hunk 2 above)
Comment (ivanpauno) [reply kinetic/distribution.yaml:9504]: mmm ... the top level `CMakeLists` is not from a ROS package, and the example you pointed out seems to be using a legacy rosbuild package manifest (which I'm not sure if catkin supports or not).

I don't know if non-ROS packages are accepted here or not. @cottsay can you chime in?

File: kinetic/distribution.yaml (hunk 3 above)
Comment (ivanpauno) [reply kinetic/distribution.yaml:17544]: thank you

**Comment by ivanpauno** [comment]
@SuperAruba can you open a PR for each of the modifications?
- multipoint_navigation_receiver
- wit-imu
- orb_slam2_with_maps-odom

I think this can be reviewed more easily in that way.

**Comment by SuperAruba** [comment]
Sorry,I don't know how to open another PR.

**Comment by ivanpauno** [comment]
> Sorry,I don't know how to open another PR.

You have to create branches in your fork, and make modifications for a single package in different branches.
You will be able to open PRs from the different branches, targeting each of the packages.

File: kinetic/distribution.yaml (hunk 2 above)
Comment (clalancette) [reply kinetic/distribution.yaml:9504]: Old rosbuild stuff we definitely don't support.  In certain circumstances, we can take in non-ROS (catkin or ament) packages, but they need special handling at release time.

But I do think this PR should be split up, there is just too much going on in a single request.  Usually we prefer one package per pull request.

Comment (SuperAruba) [reply kinetic/distribution.yaml:9504]: OK,I am trying to create a new PR. Regarding the issue of rosbuild, we will discuss it in the orb_slam2_with_maps_odom package PR.

**Comment by SuperAruba** [comment]
There is a question,I have committing two package(multipoint_navigation_receiver and wit-imu-driver)in this PR successfully,whether I need to resubmit wit-imu-driver package?

**Comment by ivanpauno** [comment]
Both this PR and https://github.com/ros/rosdistro/pull/26525/files are adding `orb_slam2_with_maps-odom` now, so I will close this one.

> There is a question,I have committing two package(multipoint_navigation_receiver and wit-imu-driver)in this PR successfully,whether I need to resubmit wit-imu-driver package?

Yes, it would be great if you can create separate PRs for `multipoint_navigation_receiver` and `wit-imu-driver`.

**Pull request closed without merging**